### DISCUSSION TITLE
Budibase DB Table editing hides information

### DISCUSSION THREAD

**Discussion by theonlyhamish** [Budibase/budibase#8274] [Help]
HI folks, new to this but loving the Budibase UI.

Im having an issue with the Budibase DB new tables, if i import a csv, the rows and columns appear but no records appear.  I have also edited a column name and it too has disappeared although the table thinks the column is still there?

**Comment by shogunpurple** [comment]
Hi @theonlyhamish could you send your app export or the CSV you are using (if the data isn't sensitive) here please so I can test?  Thanks

**Comment by theonlyhamish** [comment]
The details are not sensitive, they do not relate to real people

[Customer Details.csv](https://github.com/Budibase/budibase/files/9803562/Customer.Details.csv)

I have half fixed the small issue with renaming the column, it now appears but at the end of the other visible columns
Thanks for your help
